src/Compilers/Test/Resources/Core/SymbolsTests/NoPia/ParametersWithoutNames.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Runtime.InteropServices;
+
+
+class Program
```

Comment:
This file isn't compiled, it is there for informational purposes only. But, if you insist, I can add explicit accessibility modifier.